azure/appservice-windows/main.tf

```diff
@@ -14,7 +14,8 @@ resource "azurerm_app_service_plan" "app" {
 }
 
 resource "azurerm_app_service" "app" {
-  name                = "as-${var.project}-${var.environment}"
+  count               = length(var.type)
+  name                = "as-${var.project}-${var.type[count.index]}-${var.environment}"
   location            = local.location
```

Comment:
isn't possible to get the location from the app service plan or from the resource group?